### ISSUE TITLE
[Feature Request]: Add HTTP(S) proxy support

### ISSUE THREAD

**Issue by RafaelLinux** [vinifmor/bauh#358]
I like your project very much !!!! It is one of the best ideas I have seen since the "Ventoy" project.

In my case, applications that need to access the Internet require that the application allows to enter the data of a HTTP(S) proxy, but BAUH does not show the possibility to configure a proxy.

Would it be possible for you to add such an option in the future?

Thanks for your project.

**Comment by vinifmor** [comment]
Hi, @RafaelLinux . It's nice to see that bauh has been useful for you. Do you mean to configure a proxy for all HTTP calls made by bauh ?

**Comment by RafaelLinux** [comment]
Yes, that is. If BAUH can't use a proxy, can't be used in proxied intranets  :(
